package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "theia",
+  "name": "theia-ide",
   "version": "0.0.1-alpha.1",
   "description": "Theia - JavaScript IDE Framework",
```

Comment:
Could we also update the description to the same which we use at github: `Theia is a cloud & desktop IDE implemented in TypeScript.`